packages/loot-core/src/shared/util.ts

```diff
@@ -412,7 +412,7 @@ export function looselyParseAmount(amount: string) {
   // Look for a decimal marker, then look for either 1-2 or 5-9 decimal places.
   // This avoids matching against 3 places which may not actually be decimal
   const m = amount.match(/[.,]([^.,]{5,9}|[^.,]{1,2})$/);
-  if (!m || m.index === undefined || m.index === 0) {
+  if (!m || m.index === undefined) {
```

Comment:
This may need some vetting.  The reason for this is that the check I removed made it so a value like '.50' would return as $50.  This will return the correct amount now even if there is no non-decimal digits.